packages/editor-ui/src/components/InviteUsersModal.vue

```diff
@@ -0,0 +1,170 @@
+		<template slot="content">
+			<n8n-form-inputs
+				:inputs="config"
+				:eventBus="formBus"
+				:columnView="true"
+				@input="onInput"
+				@submit="onSubmit"
+			/>
+		</template>
+		<template slot="footer">
+			<n8n-button :loading="loading" :disabled="!enabledButton" :label="buttonLabel" @click="onSubmitClick" float="right" />
+		</template>
+	</Modal>
+</template>
+
+
+<script lang="ts">
+import mixins from "vue-typed-mixins";
+
+import { showMessage } from "@/components/mixins/showMessage";
+import Modal from "./Modal.vue";
+import Vue from "vue";
+import { IFormInputs, IUserResponse } from "@/Interface";
+import { VALID_EMAIL_REGEX, INVITE_USER_MODAL_KEY } from "@/constants";
+import { ROLE } from "@/modules/userHelpers";
+
+const NAME_EMAIL_FORMAT_REGEX = /^.* <(.*)>$/;
+
+
+function getEmail(email: string): string {
+	let parsed = email.trim();
+	if (NAME_EMAIL_FORMAT_REGEX.test(parsed)) {
+		const matches = parsed.match(NAME_EMAIL_FORMAT_REGEX);
+		if (matches && matches.length === 2) {
+			parsed = matches[1];
+		}
+	}
+	return parsed;
+}
+
+export default mixins(showMessage).extend({
+	components: { Modal },
+	name: "InviteUsersModal",
+	props: {
+		modalName: {
+			type: String,
+		},
+	},
+	data() {
+		return {
+			config: null as IFormInputs | null,
+			formBus: new Vue(),
+			modalBus: new Vue(),
+			emails: '',
+			loading: false,
+			INVITE_USER_MODAL_KEY,
+		};
+	},
+	mounted() {
+		this.config = [
+			{
+				name: 'emails',
+				properties: {
+					label: 'New user email address(es)',
+					required: true,
+					validationRules: [{name: 'VALID_EMAILS'}],
+					validators: {
+						VALID_EMAILS: {
+							validate: this.validateEmails,
+						},
+					},
+					placeholder: 'name1@email.com, name2@email.com, ...',
+				},
+			},
+			{
+				name: 'role',
+				initialValue: 'member',
+				properties: {
+					label: 'Role',
+					required: true,
+					type: 'select',
+					options: [
+						{
+							value: ROLE.Member,
+							label: 'Member',
+						},
+					],
+				},
+			},
+		];
+	},
+	computed: {
+		emailsCount(): number {
+			return this.emails.split(',').filter((email: string) => !!email.trim()).length;
+		},
+		buttonLabel(): string {
+			if (this.emailsCount > 1) {
+				return `Invite ${this.emailsCount} users`;
+			}
+
+			return 'Invite user';
+		},
+		enabledButton(): boolean {
+			return this.emailsCount >= 1;
+		},
+	},
+	methods: {
+		validateEmails: (value: string) => {
+			value.split(',').forEach((email: string) => {
+				const parsed = getEmail(email);
+
+				if (!!parsed.trim() && !VALID_EMAIL_REGEX.test(String(parsed).trim().toLowerCase())) {
+					throw new Error(`"${parsed.trim()}" is not a valid email`);
+				}
+			});
+		},
+		onInput(e: {name: string, value: string}) {
+			if (e.name === 'emails') {
+				this.emails = e.value;
+			}
+		},
+		async onSubmit() {
+			try {
+				this.loading = true;
+
+				const emails = this.emails.split(',')
+					.map((email) => ({email: getEmail(email).trim()}))
```

Comment:
Correct me if I'm wrong, but I think we already trim in the `getEmail` function

```suggestion
					.map((email) => ({ email: getEmail(email) }))
```